script/Deploy.s.sol

```diff
@@ -1,43 +1,41 @@
+// Copyright 2024 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
 // SPDX-License-Identifier: Apache-2.0
 
-pragma solidity ^0.8.17;
+pragma solidity ^0.8.19;
 
+import {BonsaiCheats} from "bonsai/BonsaiCheats.sol";
 import {Script} from "forge-std/Script.sol";
 import {console2} from "forge-std/console2.sol";
-import {IBonsaiRelay} from "bonsai/IBonsaiRelay.sol";
-import {BonsaiCheats} from "bonsai/BonsaiCheats.sol";
+import {IRiscZeroVerifier} from "bonsai/IRiscZeroVerifier.sol";
+import {ControlID, RiscZeroGroth16Verifier} from "bonsai/groth16/RiscZeroGroth16Verifier.sol";
+
+import "../contracts/BonsaiStarter.sol";
 
-import {BonsaiDeploy} from "./BonsaiDeploy.sol";
-import {BonsaiStarter} from "../contracts/BonsaiStarter.sol";
-
-/// @notice Deployment script for the BonsaiStarter project.
-/// @dev Use the following environment variables to control the deployment:
-///     * DEPLOYER_ADDRESS address of the wallet to be used for sending deploy transactions.
-///         Must be unlocked on the RPC provider node.
-///     * DEPLOYER_PRIVATE_KEY private key of the wallet to be used for deployment.
-///         Alternative to using DEPLOYER_ADDRESS.
-///     * DEPLOY_VERFIER_ADDRESS address of a predeployed IRiscZeroVerifier contract.
-///         If not specified and also DEPLOY_RELAY_ADDRESS is not specified,
-///         a new RiscZeroGroth16Verifier will be deployed.
-///     * DEPLOY_RELAY_ADDRESS address of a predeployed BonsaiRelay contract.
-///         If not specified, a new BonsaiRelay will be deployed.
-///     * DEPLOY_UPLOAD_IMAGES true or false indicating whether to upload the zkVM guest images to
-///         Bonsai. Default is false.
-///     * RISC0_DEV_MODE indicates what mode of proving is being used and decides which relay
-///         contract to deploy.
-///         * If "true": The mock BonsaiTestRelay contract will be used.
-///         * If "false" or unset: The fully verifying BonsaiRelay contract will be used.
-contract Deploy is Script, BonsaiCheats, BonsaiDeploy {
+contract EvenNumberDeploy is Script, BonsaiCheats {
```

Comment:
Could you add some documentation on this script for usage, and in particular the env vars it reads?